assets/src/pages/room/hooks/useMembraneClient.tsx

```diff
@@ -135,17 +142,15 @@ export const useMembraneClient = (
         webrtc.join(peerMetadata);
         setWebrtc(webrtc);
       })
-      .receive("error", (_response) => {
-        handleError(`Couldn't establish signaling connection`);
+      .receive("error", (response) => {
+        if (response === "server_full") {
+          handleError("Server has reached maximum allowed load. Disconnecting");
```

Comment:
The front-end team might want to consider implementing a different way of handling this error, this definitely isn't ideal

Reply:
cc: @kamil-stasiak